xla/service/gpu/transforms/cudnn_custom_call_compiler.cc

```diff
@@ -345,6 +352,23 @@ absl::StatusOr<se::gpu::CudnnGraph> BuildGraphForCustomCallToBackwardFMHA(
                       GetDNNFmhaMaskKindFromCudnnFmhaMaskKind(cudnn_mask_type));
 
   const int sliding_window_length = config.sliding_window_length();
+  auto computations = custom_call->called_computations();
+  const HloComputation *score_mod_fwd_comp = nullptr;
+  const HloComputation *score_mod_bwd_comp = nullptr;
+  if (computations.size() == 1) {
```

Comment:
I guess computations.size() > 1 would be an error? Could you please add a TF_RET_CHECK for that?